frontend/assets/scss/controls.scss

```diff
@@ -0,0 +1,66 @@
+
+@use "sass:math";
+
+.control-container {
+    display: block;
+    position: relative;
+    cursor: pointer;
+    user-select: none;
```

Comment:
Being able to select label text is standard HTML behavior. Are we removing that feature as part of the choice noted above to make the full element width selectable?

Reply:
Oops, yeah this was my blind copying from a W3Schools implementation